src/chocolatey/infrastructure.app/commands/ChocolateyExportCommand.cs

```diff
@@ -36,11 +36,14 @@ public class ChocolateyExportCommand : ICommand
     {
         private readonly INugetService _nugetService;
         private readonly IFileSystem _fileSystem;
+        private readonly IChocolateyPackageInformationService _packageInfoService;
+        private readonly IChocolateyPackageService _packageService;

```

Comment:
This is defined here, and it is used further down on line 159, but it is never instantiated via the constructor. Is this an omission, or what is the thinking here?

Reply:
It is required when instantiating the `ChocolateyInstallCommand`, but when setting up to parse install arguments (via `ChocolateyInstallCommand.configure_argument_parser`), there is no usage/check that the passed in `IChocolateyPackageService` is actually instantiated. 

If you think it would be best, I could instantiate it?